src/domain/balances/entities/schemas/balance.schema.ts

```diff
@@ -26,6 +26,8 @@ const balanceSchema: Schema = {
     tokenAddress: { type: 'string', nullable: true, default: null },
     token: { anyOf: [{ type: 'null' }, { $ref: 'balance-token.json' }] },
     balance: { type: 'string' },
+    fiatBalance: { type: 'string', nullable: true, default: null },
+    fiatConversion: { type: 'string', nullable: true, default: null },
```

Comment:
These properties were present in the domain entity but weren't included in the schema. This is probably a leftover of the previous implementation (pre-Coingecko) where these were calculated at the route level.